devel/libhv/Portfile

```
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
```

Instead of directly specifying a requirement for `openssl` in `depends_build-append`, you may actually want to use the `openssl` portgroup:

```suggestion
PortGroup           openssl 1.0

```

---

Thanks, fixed.